### ISSUE TITLE
full paths when in emacs

### ISSUE THREAD

**Issue by fommil** [scalacenter/scalafix#648]
the following is typical output when a lint catches a problem.

First thing: the format is much better than before!

Secondly, unfortunately this is not a full filename path so it does not fully support jump to error. We need the full path here and we take care of cleaning that up when presented to the user.

Can you please use the full path, either feature flagged, or when envvar `INSIDE_EMACS` is detected?

Note that my terminal, urxvt, will also struggle to support jump to file unless it is emitting a full pathname.

```
[error] src/main/scala/logic.scala:59:14: error: [DisableSyntax.keywords.null] null is disabled
      Option(null).get
             ^
```

**Comment by olafurpg** [comment]
Thanks for reporting! I agree the full path should be shown. This is in fact not an intentional attempt to prettify the output, the presented paths are relative because that's how they are persisted in semanticdbs in order to make *.semanticdb portable across machines.

The logic to map a relative path to it's original file is somewhere in `CliRunner`. The fix will be more tricky than a quick `path.toAbsolutePath` because `ScalafixOptions.sourceroot` (absolute path that can be used to absolutize relative paths in semanticdb) need to be made available to the reporter.

**Comment by fommil** [comment]
heh, ironically I have the opposite problem in https://github.com/lihaoyi/sourcecode/issues/40

**Comment by olafurpg** [comment]
Fixed in #737, trees are now parsed from an `AbsolutePath`
https://github.com/scalacenter/scalafix/blob/b6360273a5cd3054d1d61515eb144891d9d7e11c/scalafix-cli/src/main/scala/scalafix/internal/v1/MainOps.scala#L105-L106
